app/services/idv/vendor_validator.rb

```diff
@@ -25,5 +25,22 @@ def idv_agent
         vendor: (idv_session.vendor || idv_vendor.pick)
       )
     end
+
+    def result
+      @_result ||= try_submit
+    end
+
+    def try_agent_action
+      yield
+    rescue => err
+      err_msg = err.to_s
```

Comment:
I think we should also post to newrelic here? `NewRelic::Agent.notify_exception(err)` or something

Otherwise if we just eat these, we may mask outages and things like that?